librz/core/cmd_descs/cmd_yank.yaml

```diff
@@ -0,0 +1,84 @@
+# SPDX-FileCopyrightText: 2021 RizinOrg <info@rizin.re>
+# SPDX-License-Identifier: LGPL-3.0-only
+---
+name: cmd_yank
+commands:
+  - name: y
+    cname: yank
+    summary: Yank bytes / Show yank contents
+    type: RZ_CMD_DESC_TYPE_ARGV_STATE
+    modes:
+      - RZ_OUTPUT_MODE_STANDARD
+      - RZ_OUTPUT_MODE_RIZIN
+      - RZ_OUTPUT_MODE_JSON
+    args:
+      - name: len
+        type: RZ_CMD_ARG_TYPE_RZNUM
+        optional: true
+  - name: y!
+    cname: yank_editor
+    summary: Open cfg.editor to edit the clipboard
+    args: []
+  - name: yf
+    cname: yank_file
+    summary: Yank [blocksize] bytes from file
+    args:
+      - name: len
+        type: RZ_CMD_ARG_TYPE_RZNUM
+      - name: file
+        type: RZ_CMD_ARG_TYPE_FILE
+  - name: yfa
+    cname: yank_whole_file
+    summary: Yank whole file into clipboard
+    args:
+      - name: file
+        type: RZ_CMD_ARG_TYPE_FILE
+  - name: yp
```

Comment:
Did you change it on purpose from `yq` to `yp`? `yp` was useless, right?

Reply:
Also... Isn't this the same as `yx`?

Reply:
Initially I thought the same, but they produce different output. I am not sure what `yp` is, but I kept the previous output

Reply:
I'd try to review a bit the existing commands before merging this, since `y` commands are quite limited in scope it should be feasible. I'll try to help on this.

Reply:
Ok, I returned back `yq` as `y` with `RZ_OUTPUT_MODE_QUIET`, and fixed `yp` description as printing the raw data.